AprilSylph/XKit-Rewritten
# AccessKit: Make links blue consistently

### Description


This fixes instances where AccessKit's "Make links in posts blue" feature does not work (see linked issue), as well as choosing a color from the palette that is a) more often actually blue and b) sort of arguably more correct... on a larger number of palettes? (see comments below)

Resolves #1135.

### Testing steps


Test the relevant AccessKit option on:

> links that go to anywhere on a Tumblr domain, mentions... or links in posts when they're reblogged on the staff blog or Matt's blog

Test the relevant AccessKit option on various palettes; see the comments below.
